api/src/test/java/org/openmrs/api/context/TestUuidAuthenticationScheme.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.context;
+
+import org.openmrs.User;
+
+/*
+ * This test auth scheme logs a user simply based on asserting that its UUID is valid and points to an exisiting user.
+ */
+public class TestUuidAuthenticationScheme extends DaoAuthenticationScheme {
```

Comment:
Wouldn't it better to instead of use a test with a username and password? Some sort of admin/test combination. For this to me looks closer to the real basic authentication which we have that is based on a username and password combination.

Reply:
Just to say this more explicitly, this test should also stay to ensure that we support more than just our basic username and password db based authentication.

Reply:
@dkayiwa yes absolutely, this test should have existed already with the method that we suggest to deprecate. Anyway I added it in the commit #3.